presto-parser/src/main/antlr4/io/prestosql/sql/parser/SqlBase.g4

```diff
@@ -140,6 +140,10 @@ statement
     | DESCRIBE INPUT identifier                                        #describeInput
     | DESCRIBE OUTPUT identifier                                       #describeOutput
     | SET PATH pathSpecification                                       #setPath
+    | UPDATE qualifiedName
```

Comment:
I'd format this so the assignment list is together
```java
| UPDATE qualifiedName
    SET updateAssignment (',' updateAssignment)*
    (WHERE where=booleanExpression)?                               #update

Reply:
Changed.